custom_components/smartir/light.py

```diff
@@ -62,35 +52,22 @@ def __init__(self, hass: HomeAssistant, config: ConfigType, device_data):
         SmartIR.__init__(self, hass, config, device_data)
 
         self._brightness = None
-        self._colortemp = None
-
-        self._brightnesses = device_data["brightness"]
-        self._colortemps = device_data["colorTemperature"]
-
-        if CMD_COLOR_TEMPERATURE in self._commands or (
-            CMD_COLOR_MODE_COLDER in self._commands
-            and CMD_COLOR_MODE_WARMER in self._commands
-        ):
-            self._colortemp = self.max_color_temp_kelvin
-
-        if (
-            CMD_NIGHTLIGHT in self._commands
-            or CMD_BRIGHTNESS in self._commands
-            or (
-                CMD_BRIGHTNESS_INCREASE in self._commands
-                and CMD_BRIGHTNESS_DECREASE in self._commands
-            )
-        ):
-            self._brightness = 100
-            self._support_brightness = True
-        else:
-            self._support_brightness = False
+        self._color_temp = None
+
+        self._brightness_list = device_data.get("brightness")
+        self._color_temp_list = device_data.get("colorTemperatures")
```

Comment:
It is "colorTemperature" in the configs, and in the device_data.py checking above.

Reply:
Thank you, good catch. fixed in the last commit!